File: zad3/app/src/main/java/com/example/zad_2/ui/page4/FourthViewModel.java
package com.example.zad_2.ui.page4;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class FourthViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public FourthViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Co mogę jeść? Wybierz przepis...");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: zad4/app/src/main/java/com/example/zad4/ui/page4/FourthViewModel.kt
package com.example.zad4.ui.page4

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class FourthViewModel : ViewModel() {
    private val mText: MutableLiveData<String?>

    init {
        mText = MutableLiveData()
        mText.value = "Co mogę jeść? Wybierz przepis..."
    }

    val text: LiveData<String?>
        get() = mText
}